src/plugins/modifiers.ts

```diff
@@ -159,6 +160,45 @@ export class Modifiers {
                 }
             }
         }
+        // add NameMapsTo after modifier and if generic update exists, set the setter_arg_name
+        this.codeModel.operationGroups.forEach(operationGroup => {
+            let operations = operationGroup.operations;
+            operations.forEach(operation => {
+                let listCnt = 0;
+                let param = null;
+                operation.parameters.forEach(parameter => {
+                    if(!isNullOrUndefined(parameter.language['az'])) {
+                        parameter.language['az']['mapsto'] = parameter.language['az']['name'].replace(/-/g, '_');
+                        if(operation.language['az'].name.endsWith("create") && parameter['flattened'] != true) {
```

Comment:
>endsWith [](start = 57, length = 8)

why endWith?

Reply:
because the az name is subclass + " " + "create" like `mysql create`. which is consistent the first solution of polymorphism